src/bootstrap/src/core/config/config.rs

```diff
@@ -1904,6 +1912,21 @@ impl Config {
             config.ferrocene_ignore_document_signatures =
                 f.ignore_document_signatures.unwrap_or(false);
             config.ferrocene_technical_report_url = f.technical_report_url;
+
+            config.ferrocene_test_outcomes = match (f.test_outcomes.as_deref(), f.test_outcomes_dir)
+            {
+                (None | Some("disabled"), None) => FerroceneTestOutcomes::Disabled,
+                (Some("download-ci"), None) => FerroceneTestOutcomes::DownloadCi,
+                (Some("custom"), Some(path)) => FerroceneTestOutcomes::Custom(path.into()),
+                // Legacy: allow setting test-outcomes-dir without test-outcomes to avoid breaking
+                // developers currently setting it, only if test-outcomes is not configured.
+                (None, Some(path)) => FerroceneTestOutcomes::Custom(path.into()),
```

Comment:
The only developers using this config at the moment are we, or are there others? So maybe we don't need the legacy option?

Reply:
I'd like to keep this for a small transition period to avoid just breaking the local `config.toml` of folks. We can open an issue to clean this up.